content/en/docs/pipelines/sdk/python-function-components.ipynb

```diff
@@ -27,7 +27,8 @@
     "\n",
```

Comment:
<blockquote>Your function must install the package. For example, your function can use</blockquote><blockquote>the [<code>subprocess</code> module][subprocess] to run a command like <code>pip install</code>&nbsp;that installs a package.</blockquote><p>Do we need to call out this requires <code>pip</code> to be available in the container? This is true for a various Python based image like <code>python</code> or <code>tensorflow</code> but not always true.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/website/pull/2318/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
In this case, I don't think that it is necessary. It specifies that you can use a command "like pip", so users still need to evaluate what the correct option is for their container.